components/tools/OmeroPy/src/omero/plugins/web.py

```diff
@@ -339,7 +366,13 @@ def start(self, args):
         link = ("%s:%s" % (settings.APPLICATION_SERVER_HOST,
                            settings.APPLICATION_SERVER_PORT))
         location = self._get_python_dir() / "omeroweb"
-        self.ctx.out("Starting OMERO.web... ", newline=False)
+        deploy = getattr(settings, 'APPLICATION_SERVER')
+        if deploy == settings.WSGI:
+            self.ctx.out("OMERO.web is ready. Restart apache.")
```

Comment:
@manics adding new config requires reload. I will remove startup in favor of syncmedia and session cleanup only, otherwise nothing else is needed.